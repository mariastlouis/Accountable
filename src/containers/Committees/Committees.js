import React from 'react';
import PropTypes from 'prop-types';
import './Committees.css';

const Committees = ({committees}) => {


  const capitalize = (string) =>{
    return string.charAt(0).toUpperCase() + string.slice(1);
  };

  const mapCommittee = (committees) => {
    if (committees) {
      const committeeKeys = Object.keys(committees).map((committee, index) =>{
        if (committees[committee].name !== 'none') {
          return (
            
              <li className = "committee-element"key = {index}> <span className = "darker">{capitalize(committees[committee].position)} </span>
             of <a href = {committees[committee].website}> {committees[committee].name} </a></li>
          
          );
        }
      });
      return committeeKeys; 
    }
  };

  return (
    <div className = "committees">
      <div className = "committee-hed">
        <h2> Committee Assignments </h2>
      </div>
      <div className = "committee-info">
        <ul>
        {mapCommittee(committees)}
        </ul>
      </div>
      <hr />
    </div>

  );
};  

export default Committees;

Committees.propTypes = {
  committees: PropTypes.array
};