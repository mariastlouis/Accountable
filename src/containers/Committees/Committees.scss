@import '../../assets/variable.scss';

.committee-hed {
  display: flex;
  justify-content: center;
}

.committees {
  background: #eaeef1;
  max-width: 800px;
  margin-left: auto;
  margin-right: auto;
  padding: 20px;
}


.committee-info {
  display: flex;
  align-items: center;
  justify-content: center;
}

.darker {
  font-weight: 600;
}

.committee-element {
  margin-bottom: 15px;
}