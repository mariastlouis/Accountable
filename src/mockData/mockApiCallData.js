export const mockLawmakerApiResponse = 
    {
        "last_name": "Duran",
        "updated_at": "2017-12-20 07:08:21",
        "full_name": "Crisanta Duran",
        "id": "COL000067",
        "first_name": "Crisanta",
        "middle_name": "",
        "district": "5",
        "chamber": "lower",
        "state": "co",
        "votesmart_id": "125477",
        "party": "Democratic",
        "suffixes": "",
        "email": "",
        "all_ids": [
            "COL000067"
        ],
        "leg_id": "COL000067",
        "active": true,
        "transparencydata_id": "75bb4e6db9f946caa09e9d84256b85a8",
        "photo_url": "http://leg.colorado.gov/sites/default/files/styles/width_300/public/2017b_h-speaker%20duran%20-17.jpg?itok=573JhyU2",
        "level": "state",
        "url": "http://leg.colorado.gov/legislators/crisanta-duran",
        "country": "us",
        "created_at": "2012-01-19 01:01:49",
        "+occupation": "Attorney",
        "offices": [
            {
                "fax": null,
                "name": "Capitol Office",
                "phone": null,
                "address": "200 E. Colfax\nRM 307\nDenver, CO 80203",
                "type": "capitol",
                "email": null
            }
        ],
        "+phone": "303 866 2925"
    }


export const mockLawmakerObject = {
    id: "COL000067",
    contact: {
        firstName: "Crisanta",
        lastName: "Duran",
        party: "Democratic",
        image: "http://leg.colorado.gov/sites/default/files/styles/width_300/public/2017b_h-speaker%20duran%20-17.jpg?itok=573JhyU2",
        website: "http://leg.colorado.gov/legislators/crisanta-duran",
        phone: "303 866 2925",
        active: true,
        occuption: "Attorney",
        chamber: "lower",
        district:"5"
    }
}

export const mockDetailApiCall = {
  "last_name": "Duran",
  "updated_at": "2017-12-22 07:08:43",
  "sources": [
    {
      "url": "http://leg.colorado.gov/legislators/crisanta-duran"
    },
    {
      "url": "http://leg.colorado.gov/legislators"
    }
  ],
  "full_name": "Crisanta Duran",
  "old_roles": {
    "2013-2014": [
      {
        "term": "2013-2014",
        "end_date": null,
        "district": "5",
        "chamber": "lower",
        "state": "co",
        "party": "Democratic",
        "type": "member",
        "start_date": null
      }
    ],
    "2011-2012": [
      {
        "term": "2011-2012",
        "end_date": null,
        "district": "5",
        "chamber": "lower",
        "state": "co",
        "party": "Democratic",
        "type": "member",
        "start_date": null
      },
      {
        "term": "2011-2012",
        "committee_id": "COC000009",
        "end_date": null,
        "chamber": "lower",
        "state": "co",
        "committee": "Judiciary",
        "position": "member",
        "type": "committee member",
        "start_date": null
      },
      {
        "term": "2011-2012",
        "committee_id": "COC000022",
        "end_date": null,
        "chamber": "lower",
        "state": "co",
        "committee": "State, Veterans, & Military Affairs",
        "position": "member",
        "type": "committee member",
        "start_date": null
      },
      {
        "term": "2011-2012",
        "committee_id": "COC000015",
        "end_date": null,
        "chamber": "lower",
        "state": "co",
        "committee": "Legislative Council",
        "position": "member",
        "type": "committee member",
        "start_date": null
      }
    ],
    "2015-2016": [
      {
        "term": "2015-2016",
        "end_date": null,
        "district": "5",
        "chamber": "lower",
        "state": "co",
        "party": "Democratic",
        "type": "member",
        "start_date": null
      },
      {
        "term": "2015-2016",
        "committee_id": "COC000122",
        "chamber": "joint",
        "state": "co",
        "subcommittee": null,
        "committee": "Legislative Council",
        "position": "member",
        "type": "committee member"
      },
      {
        "term": "2015-2016",
        "committee_id": "COC000147",
        "chamber": "joint",
        "state": "co",
        "subcommittee": null,
        "committee": "Executive Committee of the Legislative Council",
        "position": "member",
        "type": "committee member"
      }
    ]
  },
  "id": "COL000067",
  "first_name": "Crisanta",
  "middle_name": "",
  "district": "5",
  "chamber": "lower",
  "state": "co",
  "votesmart_id": "125477",
  "party": "Democratic",
  "suffixes": "",
  "email": "",
  "all_ids": [
    "COL000067"
  ],
  "leg_id": "COL000067",
  "active": true,
  "transparencydata_id": "75bb4e6db9f946caa09e9d84256b85a8",
  "photo_url": "http://leg.colorado.gov/sites/default/files/styles/width_300/public/2017b_h-speaker%20duran%20-17.jpg?itok=573JhyU2",
  "roles": [
    {
      "term": "2017-2018",
      "end_date": null,
      "district": "5",
      "chamber": "lower",
      "state": "co",
      "party": "Democratic",
      "type": "member",
      "start_date": null
    }
  ],
  "level": "state",
  "url": "http://leg.colorado.gov/legislators/crisanta-duran",
  "country": "us",
  "created_at": "2012-01-19 01:01:49",
  "+occupation": "Attorney",
  "offices": [
    {
      "fax": null,
      "name": "Capitol Office",
      "phone": null,
      "address": "200 E. Colfax\nRM 307\nDenver, CO 80203",
      "type": "capitol",
      "email": null
    }
  ],
  "+phone": "303 866 2925"
}

export const mockLatLongApiCall = [
  {
    "last_name": "Sias",
    "updated_at": "2018-01-06 07:09:05",
    "sources": [
      {
        "url": "http://leg.colorado.gov/legislators/lang-sias"
      },
      {
        "url": "http://leg.colorado.gov/legislators"
      }
    ],
    "full_name": "Sias, Lang",
    "old_roles": {
      "2015-2016": [
        {
          "term": "2015-2016",
          "end_date": null,
          "district": "27",
          "chamber": "lower",
          "state": "co",
          "party": "Republican",
          "type": "member",
          "start_date": null
        },
        {
          "term": "2015-2016",
          "committee_id": "COC000150",
          "chamber": "joint",
          "state": "co",
          "subcommittee": null,
          "committee": "Colorado Health Insurance Exchange Oversight Committee",
          "position": "member",
          "type": "committee member"
        },
        {
          "term": "2015-2016",
          "committee_id": "COC000126",
          "chamber": "lower",
          "state": "co",
          "subcommittee": null,
          "committee": "Business Affairs and Labor",
          "position": "member",
          "type": "committee member"
        },
        {
          "term": "2015-2016",
          "committee_id": "COC000152",
          "chamber": "joint",
          "state": "co",
          "subcommittee": null,
          "committee": "Exchange Review Committee",
          "position": "member",
          "type": "committee member"
        },
        {
          "term": "2015-2016",
          "committee_id": "COC000134",
          "chamber": "lower",
          "state": "co",
          "subcommittee": null,
          "committee": "Finance",
          "position": "member",
          "type": "committee member"
        }
      ]
    },
    "id": "COL000198",
    "first_name": "Lang",
    "middle_name": "",
    "district": "27",
    "chamber": "lower",
    "state": "co",
    "boundary_id": "ocd-division/country:us/state:co/sldl:27",
    "email": "lang.sias.house@state.co.us",
    "all_ids": [
      "COL000198"
    ],
    "leg_id": "COL000198",
    "party": "Republican",
    "active": true,
    "photo_url": "http://leg.colorado.gov/sites/default/files/styles/width_300/public/2017b_rsz_sias-co-17.jpg?itok=mWTzJy_q",
    "roles": [
      {
        "term": "2017-2018",
        "end_date": null,
        "district": "27",
        "chamber": "lower",
        "state": "co",
        "party": "Republican",
        "type": "member",
        "start_date": null
      },
      {
        "term": "2017-2018",
        "committee_id": "COC000127",
        "chamber": "lower",
        "state": "co",
        "subcommittee": null,
        "committee": "Education",
        "position": "member",
        "type": "committee member"
      },
      {
        "term": "2017-2018",
        "committee_id": "COC000126",
        "chamber": "lower",
        "state": "co",
        "subcommittee": null,
        "committee": "Business Affairs and Labor",
        "position": "member",
        "type": "committee member"
      }
    ],
    "url": "http://leg.colorado.gov/legislators/lang-sias",
    "created_at": "2015-02-04 01:30:44",
    "+occupation": "Commercial Pilot",
    "offices": [
      {
        "fax": null,
        "name": "Capitol Office",
        "phone": "303-866-2962",
        "address": "200 E. Colfax\nRM 307\nDenver, CO 80203",
        "type": "capitol",
        "email": "lang.sias.house@state.co.us"
      }
    ],
    "suffixes": ""
  },
  {
    "last_name": "Zenzinger",
    "updated_at": "2018-01-06 07:09:05",
    "sources": [
      {
        "url": "http://leg.colorado.gov/legislators/rachel-zenzinger"
      },
      {
        "url": "http://leg.colorado.gov/legislators"
      }
    ],
    "full_name": "Zenzinger, Rachel",
    "id": "COL000229",
    "first_name": "Rachel",
    "middle_name": "",
    "district": "19",
    "chamber": "upper",
    "state": "co",
    "boundary_id": "ocd-division/country:us/state:co/sldu:19",
    "email": "senatorrachelz@gmail.com",
    "all_ids": [
      "COL000229"
    ],
    "leg_id": "COL000229",
    "party": "Democratic",
    "active": true,
    "photo_url": "http://leg.colorado.gov/sites/default/files/styles/width_300/public/2017b_rsz_zenzinger-co-17.jpg?itok=MOpIWzSu",
    "roles": [
      {
        "term": "2017-2018",
        "end_date": null,
        "district": "19",
        "chamber": "upper",
        "state": "co",
        "party": "Democratic",
        "type": "member",
        "start_date": null
      },
      {
        "term": "2017-2018",
        "committee_id": "COC000197",
        "chamber": "upper",
        "state": "co",
        "subcommittee": null,
        "committee": "Legislative Interim Committee on School Finance",
        "position": "member",
        "type": "committee member"
      },
      {
        "term": "2017-2018",
        "committee_id": "COC000214",
        "chamber": "upper",
        "state": "co",
        "subcommittee": null,
        "committee": "Statutory Revision Committee",
        "position": "member",
        "type": "committee member"
      },
      {
        "term": "2017-2018",
        "committee_id": "COC000206",
        "chamber": "upper",
        "state": "co",
        "subcommittee": null,
        "committee": "Transportation Legislation Review Committee",
        "position": "member",
        "type": "committee member"
      },
      {
        "term": "2017-2018",
        "committee_id": "COC000203",
        "chamber": "upper",
        "state": "co",
        "subcommittee": null,
        "committee": "School Safety and Youth in Crisis Committee",
        "position": "member",
        "type": "committee member"
      },
      {
        "term": "2017-2018",
        "committee_id": "COC000136",
        "chamber": "upper",
        "state": "co",
        "subcommittee": null,
        "committee": "Transportation",
        "position": "member",
        "type": "committee member"
      }
    ],
    "url": "http://leg.colorado.gov/legislators/rachel-zenzinger",
    "created_at": "2017-01-13 07:06:33",
    "+occupation": "Education",
    "offices": [
      {
        "fax": null,
        "name": "Capitol Office",
        "phone": "303-866-4840",
        "address": "200 E. Colfax\nRM 346\nDenver, CO 80203",
        "type": "capitol",
        "email": "senatorrachelz@gmail.com"
      }
    ],
    "suffixes": ""
  }
]
