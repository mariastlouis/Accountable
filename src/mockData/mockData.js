export const mockLawmakerApiResponse = [
    {
        "last_name": "Duran",
        "updated_at": "2017-12-20 07:08:21",
        "full_name": "Crisanta Duran",
        "id": "COL000067",
        "first_name": "Crisanta",
        "middle_name": "",
        "district": "5",
        "chamber": "lower",
        "state": "co",
        "votesmart_id": "125477",
        "party": "Democratic",
        "suffixes": "",
        "email": "",
        "all_ids": [
            "COL000067"
        ],
        "leg_id": "COL000067",
        "active": true,
        "transparencydata_id": "75bb4e6db9f946caa09e9d84256b85a8",
        "photo_url": "http://leg.colorado.gov/sites/default/files/styles/width_300/public/2017b_h-speaker%20duran%20-17.jpg?itok=573JhyU2",
        "level": "state",
        "url": "http://leg.colorado.gov/legislators/crisanta-duran",
        "country": "us",
        "created_at": "2012-01-19 01:01:49",
        "+occupation": "Attorney",
        "offices": [
            {
                "fax": null,
                "name": "Capitol Office",
                "phone": null,
                "address": "200 E. Colfax\nRM 307\nDenver, CO 80203",
                "type": "capitol",
                "email": null
            }
        ],
        "+phone": "303 866 2925"
    }
]

export const mockLawmakerObject = {
    id: "COL000067",
    contact: {
        firstName: "Crisanta",
        lastName: "Duran",
        party: "Democratic",
        image: "http://leg.colorado.gov/sites/default/files/styles/width_300/public/2017b_h-speaker%20duran%20-17.jpg?itok=573JhyU2",
        website: "http://leg.colorado.gov/legislators/crisanta-duran",
        phone: "303 866 2925",
        active: true,
        occuption: "Attorney",
        chamber: "lower",
        district:"5"
    }
}

export const mockDetailApiCall {
  "last_name": "Duran",
  "updated_at": "2017-12-22 07:08:43",
  "sources": [
    {
      "url": "http://leg.colorado.gov/legislators/crisanta-duran"
    },
    {
      "url": "http://leg.colorado.gov/legislators"
    }
  ],
  "full_name": "Crisanta Duran",
  "old_roles": {
    "2013-2014": [
      {
        "term": "2013-2014",
        "end_date": null,
        "district": "5",
        "chamber": "lower",
        "state": "co",
        "party": "Democratic",
        "type": "member",
        "start_date": null
      }
    ],
    "2011-2012": [
      {
        "term": "2011-2012",
        "end_date": null,
        "district": "5",
        "chamber": "lower",
        "state": "co",
        "party": "Democratic",
        "type": "member",
        "start_date": null
      },
      {
        "term": "2011-2012",
        "committee_id": "COC000009",
        "end_date": null,
        "chamber": "lower",
        "state": "co",
        "committee": "Judiciary",
        "position": "member",
        "type": "committee member",
        "start_date": null
      },
      {
        "term": "2011-2012",
        "committee_id": "COC000022",
        "end_date": null,
        "chamber": "lower",
        "state": "co",
        "committee": "State, Veterans, & Military Affairs",
        "position": "member",
        "type": "committee member",
        "start_date": null
      },
      {
        "term": "2011-2012",
        "committee_id": "COC000015",
        "end_date": null,
        "chamber": "lower",
        "state": "co",
        "committee": "Legislative Council",
        "position": "member",
        "type": "committee member",
        "start_date": null
      }
    ],
    "2015-2016": [
      {
        "term": "2015-2016",
        "end_date": null,
        "district": "5",
        "chamber": "lower",
        "state": "co",
        "party": "Democratic",
        "type": "member",
        "start_date": null
      },
      {
        "term": "2015-2016",
        "committee_id": "COC000122",
        "chamber": "joint",
        "state": "co",
        "subcommittee": null,
        "committee": "Legislative Council",
        "position": "member",
        "type": "committee member"
      },
      {
        "term": "2015-2016",
        "committee_id": "COC000147",
        "chamber": "joint",
        "state": "co",
        "subcommittee": null,
        "committee": "Executive Committee of the Legislative Council",
        "position": "member",
        "type": "committee member"
      }
    ]
  },
  "id": "COL000067",
  "first_name": "Crisanta",
  "middle_name": "",
  "district": "5",
  "chamber": "lower",
  "state": "co",
  "votesmart_id": "125477",
  "party": "Democratic",
  "suffixes": "",
  "email": "",
  "all_ids": [
    "COL000067"
  ],
  "leg_id": "COL000067",
  "active": true,
  "transparencydata_id": "75bb4e6db9f946caa09e9d84256b85a8",
  "photo_url": "http://leg.colorado.gov/sites/default/files/styles/width_300/public/2017b_h-speaker%20duran%20-17.jpg?itok=573JhyU2",
  "roles": [
    {
      "term": "2017-2018",
      "end_date": null,
      "district": "5",
      "chamber": "lower",
      "state": "co",
      "party": "Democratic",
      "type": "member",
      "start_date": null
    }
  ],
  "level": "state",
  "url": "http://leg.colorado.gov/legislators/crisanta-duran",
  "country": "us",
  "created_at": "2012-01-19 01:01:49",
  "+occupation": "Attorney",
  "offices": [
    {
      "fax": null,
      "name": "Capitol Office",
      "phone": null,
      "address": "200 E. Colfax\nRM 307\nDenver, CO 80203",
      "type": "capitol",
      "email": null
    }
  ],
  "+phone": "303 866 2925"
}
