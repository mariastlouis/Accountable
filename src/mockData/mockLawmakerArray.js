export const mockLawmakerArray = [
{
  id: 'COL000043',
  isSelected: false,
  contact: {
    firstName: 'Jon',
    lastName: 'Becker',
    party: 'Republican',
    image: "http://leg.colorado.gov/sites/default/files/styles/width_300/public/2017b_rsz_beckerjon-co-17.jpg?itok=MZ7fny_6",
    website: "http://leg.colorado.gov/legislators/jon-becker",
    phone: "303 866 2906",
    active: true,
    occupation: "Telecom Executive",
    chamber: 'lower',
    district: '65'
    }
  },
  {
  id: 'COL000067',
  isSelected: false,
  contact: {
    firstName: "Crisanta",
    lastName: 'Duran',
    party: 'Democratic',
    image: "http://leg.colorado.gov/sites/default/files/styles/width_300/public/2017b_h-speaker%20duran%20-17.jpg?itok=573JhyU2",
    website: "http://leg.colorado.gov/legislators/crisanta-duran",
    phone: "303 866 2925",
    active: true,
    occupation: "Attorney",
    chamber: 'lower',
    district: '5'
  }
}
]
