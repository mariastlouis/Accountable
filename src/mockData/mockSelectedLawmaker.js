/* eslint-disable */

export const mockSelectedLawmaker = {
  id: 'COL000033',
  contact: {
    firstName: "Lucia",
    lastName: "Guzman",
    party: "Democrat",
    image: "http://leg.colorado.gov/sites/default/files/styles/width_300/public/2017b_rsz_guzman-co-17.jpg?itok=kF6sK0P4",
    website: "http://leg.colorado.gov/legislators/lucia-guzman",
    phone: "303 866 4862",
    active: true,
    occupation: "Senator, District 34",
    chamber: "Senate",
    district: 34
  },
  committees: [
    {
      name: "none",
      position: "none",
      committeeId: "none",
      chamber: "upper",
      term: "2017-2018",
      website: "none"
    },

    {
      name: "Executive Committee of the Legislative Council",
      position: "member",
      committeeId: "COC000219",
      chamber: "upper",
      term: "2017-2018",
      website: "http://leg.colorado.gov/committees/executive-committee-legislative-council/2017-regular-session"
    }
  ],
            
  bills: [
    {
      billTitle: "Taxation Of Retail Marijuana Sales",
      billId: "COB00005158",
      billTitleId: "SB 17B-001",
      session: "2017B",
      signed: {
        signAction: "Senate Committee on Transportation Postpone Indefinitely",
        signDate: "2017-10-02 06:00:00"
      }
    },
    {
      billTitle: "Domestic Violence Fatality Review Board",
      billId: "COB00004620",
      billTitleId: "SB 17-126",
      session: "2017A",
      signed: {
        signAction: "Governor Signed",
        signDate: "2017-06-08 06:00:00"
      }
    }
  ]
  
};

export const mockSignedBill = [{
  billTitle: "Domestic Violence Fatality Review Board",
  billId: "COB00005158",
  billTitleId: "SB 17B-001",
  session: "2017B",
  signed: {
    signAction: "Governor Signed",
    signDate: "2017-10-02 06:00:00"
  }
}];

export const mockUnsignedBill = [{
  billTitle: "Taxation Of Retail Marijuana Sales",
  billId: "COB00005158",
  billTitleId: "SB 17B-001",
  session: "2017B",
  signed: {
    signAction: "Senate Committee on Transportation Postpone Indefinitely",
    signDate: "2017-10-02 06:00:00"
  }
}];
