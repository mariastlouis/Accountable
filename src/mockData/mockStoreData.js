import { mockLawmakerArray } from './mockLawmakerArray.js';
import { mockSelectedLawmaker } from './mockSelectedLawmaker.js';

export const mockStoreData = {
  lawmakers: {
  isSelected:  mockSelectedLawmaker ,
  lawmakers:  mockLawmakerArray
  }
}

