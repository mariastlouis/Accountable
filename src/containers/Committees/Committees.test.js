import  Committees  from './Committees.js';
import React from 'react';
import { shallow } from 'enzyme';
import { mockSelectedLawmaker } from '../../mockData/mockSelectedLawmaker';

describe('Committee component tests', () => {
  let mockLawmakerDetails = mockSelectedLawmaker;
  let renderedComponent;

  beforeEach(() => {
    renderedComponent = shallow(
      <Committees committees = {mockLawmakerDetails.committees}/>);
  });

  it('should render', () =>{
    expect(renderedComponent).toBeDefined();
  });

  it('should have a class of committees and committee head', () => {
    expect(renderedComponent.find('.committees').length).toEqual(1);
    expect(renderedComponent.find('.committee-hed').length).toEqual(1);
  });

  it('should match the snapshot', () => {
    expect(renderedComponent).toMatchSnapshot();
  });

  it('should capitalize the committee position when given a lowercase position', () => {
    expect(renderedComponent.contains(
      'Member')).toEqual(true);
  });


});

