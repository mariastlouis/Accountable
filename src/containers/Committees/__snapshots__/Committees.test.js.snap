// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Committee component tests should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Committees
    committees={
        Array [
            Object {
              "chamber": "upper",
              "committeeId": "none",
              "name": "none",
              "position": "none",
              "term": "2017-2018",
              "website": "none",
            },
            Object {
              "chamber": "upper",
              "committeeId": "COC000219",
              "name": "Executive Committee of the Legislative Council",
              "position": "member",
              "term": "2017-2018",
              "website": "http://leg.colorado.gov/committees/executive-committee-legislative-council/2017-regular-session",
            },
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="committee-hed"
>
          <h2>
                     Committee Assignments 
          </h2>
</div>,
        <div
          className="committee-info"
>
          <div>
                    <p>
                               
                              <span
                                        className="darker"
                              >
                                        Member
                                         
                              </span>
                              of 
                              <a
                                        href="http://leg.colorado.gov/committees/executive-committee-legislative-council/2017-regular-session"
                              >
                                         
                                        Executive Committee of the Legislative Council
                                         
                              </a>
                    </p>
          </div>
</div>,
        <hr />,
      ],
      "className": "committees",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h2>
             Committee Assignments 
</h2>,
          "className": "committee-hed",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": " Committee Assignments ",
          },
          "ref": null,
          "rendered": " Committee Assignments ",
          "type": "h2",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            undefined,
            <div>
              <p>
                             
                            <span
                                          className="darker"
                            >
                                          Member
                                           
                            </span>
                            of 
                            <a
                                          href="http://leg.colorado.gov/committees/executive-committee-legislative-council/2017-regular-session"
                            >
                                           
                                          Executive Committee of the Legislative Council
                                           
                            </a>
              </p>
</div>,
          ],
          "className": "committee-info",
        },
        "ref": null,
        "rendered": Array [
          undefined,
          Object {
            "instance": null,
            "key": "1",
            "nodeType": "host",
            "props": Object {
              "children": <p>
                 
                <span
                                className="darker"
                >
                                Member
                                 
                </span>
                of 
                <a
                                href="http://leg.colorado.gov/committees/executive-committee-legislative-council/2017-regular-session"
                >
                                 
                                Executive Committee of the Legislative Council
                                 
                </a>
</p>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  " ",
                  <span
                    className="darker"
>
                    Member
                     
</span>,
                  "of ",
                  <a
                    href="http://leg.colorado.gov/committees/executive-committee-legislative-council/2017-regular-session"
>
                     
                    Executive Committee of the Legislative Council
                     
</a>,
                ],
              },
              "ref": null,
              "rendered": Array [
                " ",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      "Member",
                      " ",
                    ],
                    "className": "darker",
                  },
                  "ref": null,
                  "rendered": Array [
                    "Member",
                    " ",
                  ],
                  "type": "span",
                },
                "of ",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      " ",
                      "Executive Committee of the Legislative Council",
                      " ",
                    ],
                    "href": "http://leg.colorado.gov/committees/executive-committee-legislative-council/2017-regular-session",
                  },
                  "ref": null,
                  "rendered": Array [
                    " ",
                    "Executive Committee of the Legislative Council",
                    " ",
                  ],
                  "type": "a",
                },
              ],
              "type": "p",
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "hr",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="committee-hed"
>
            <h2>
                         Committee Assignments 
            </h2>
</div>,
          <div
            className="committee-info"
>
            <div>
                        <p>
                                     
                                    <span
                                                className="darker"
                                    >
                                                Member
                                                 
                                    </span>
                                    of 
                                    <a
                                                href="http://leg.colorado.gov/committees/executive-committee-legislative-council/2017-regular-session"
                                    >
                                                 
                                                Executive Committee of the Legislative Council
                                                 
                                    </a>
                        </p>
            </div>
</div>,
          <hr />,
        ],
        "className": "committees",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h2>
               Committee Assignments 
</h2>,
            "className": "committee-hed",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": " Committee Assignments ",
            },
            "ref": null,
            "rendered": " Committee Assignments ",
            "type": "h2",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              undefined,
              <div>
                <p>
                                 
                                <span
                                                className="darker"
                                >
                                                Member
                                                 
                                </span>
                                of 
                                <a
                                                href="http://leg.colorado.gov/committees/executive-committee-legislative-council/2017-regular-session"
                                >
                                                 
                                                Executive Committee of the Legislative Council
                                                 
                                </a>
                </p>
</div>,
            ],
            "className": "committee-info",
          },
          "ref": null,
          "rendered": Array [
            undefined,
            Object {
              "instance": null,
              "key": "1",
              "nodeType": "host",
              "props": Object {
                "children": <p>
                   
                  <span
                                    className="darker"
                  >
                                    Member
                                     
                  </span>
                  of 
                  <a
                                    href="http://leg.colorado.gov/committees/executive-committee-legislative-council/2017-regular-session"
                  >
                                     
                                    Executive Committee of the Legislative Council
                                     
                  </a>
</p>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    " ",
                    <span
                      className="darker"
>
                      Member
                       
</span>,
                    "of ",
                    <a
                      href="http://leg.colorado.gov/committees/executive-committee-legislative-council/2017-regular-session"
>
                       
                      Executive Committee of the Legislative Council
                       
</a>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  " ",
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        "Member",
                        " ",
                      ],
                      "className": "darker",
                    },
                    "ref": null,
                    "rendered": Array [
                      "Member",
                      " ",
                    ],
                    "type": "span",
                  },
                  "of ",
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        " ",
                        "Executive Committee of the Legislative Council",
                        " ",
                      ],
                      "href": "http://leg.colorado.gov/committees/executive-committee-legislative-council/2017-regular-session",
                    },
                    "ref": null,
                    "rendered": Array [
                      " ",
                      "Executive Committee of the Legislative Council",
                      " ",
                    ],
                    "type": "a",
                  },
                ],
                "type": "p",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "hr",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
